crates/services/compression/src/service.rs

```diff
@@ -282,6 +357,25 @@ mod tests {
         InMemoryStorage::default().into_transaction()
     }
 
+    impl LatestHeight for MockStorage {
+        fn latest_height(&self) -> Option<u32> {
+            // get changes for the compressed blocks column
+            let compressed_block_changes = self.changes().get(
+                &MerkleizedColumn::<storage::column::CompressionColumn>::TableColumn(
+                    storage::column::CompressionColumn::CompressedBlocks,
+                )
+                .as_u32(),
+            );
+            compressed_block_changes.map(|changes| {
+                u32::from_be_bytes(
+                    changes.iter().last().unwrap().0.as_slice()[..4]
+                        .try_into()
+                        .unwrap(),
+                )
+            })
```

Comment:
not proud of this trick, but hey, it works

Reply:
You can just use `storage::<CompressedBlocks>()` and work with the `StorageInspect` interface

Reply:
Pushed fixed version https://github.com/FuelLabs/fuel-core/pull/2923/commits/d13fc1f01d5c4e535f9470d48372cb57ff04268c